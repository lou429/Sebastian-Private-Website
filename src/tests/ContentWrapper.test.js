import { render } from '@testing-library/react';
import ContentWrapper from '../components/contentwrapper';

test('Content wrapper snapshot', () => {
    const { container } = render(
        <ContentWrapper heading="heading">
            <p>text</p>
        </ContentWrapper>
    );

    expect(container).toMatchSnapshot();
})

test('Does content wrapper render', () => {
    let heading = "render test heading";
    let text = "render test content";

    const { getByText } = render(
        <ContentWrapper heading={heading}>
            <p>{text}</p>
        </ContentWrapper>
    );

    expect(getByText(text)).toBeInTheDocument();
    expect(getByText(heading)).toBeInTheDocument();
});