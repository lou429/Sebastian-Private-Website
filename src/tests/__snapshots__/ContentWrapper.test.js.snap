// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Content wrapper snapshot 1`] = `
<div>
  <div
    class="content-container"
  >
    <div
      class="content-heading"
    >
      <h1>
        heading
      </h1>
    </div>
    <div
      class="content-body"
    >
      <p>
        text
      </p>
    </div>
  </div>
</div>
`;
